core/templates/dev/head/css/oppia.css

```diff
@@ -3441,13 +3441,12 @@ md-card.preview-conversation-skin-supplemental-card {
   border-radius: 0;
   color: #fff;
   font-family: "Capriola", "Roboto", Arial, sans-serif;
-  font-size: 18px;
+  font-size: 14px;
   margin-top: -20px;
-  padding: 10px 0;
+  padding: 10px;
   position: absolute;
   text-transform: uppercase;
   transform: translateX(-50%);
-  width: 360px;
```

Comment:
@shaz13 Mark's reply below indicates that he would prefer that we keep the hardcoded width and change the font size. Push another commit to your branch that tries out that solution.